assets/src/modules/config/BaseLayer.js

```diff
@@ -353,6 +357,23 @@ export class WmtsBaseLayerConfig extends BaseLayerConfig {
         }
 
         super('wmts', name, cfg, wmtsProperties, wmtsOptionalProperties);
+
+        // Remove unnecessary parameters
+        let wmtsUrl = new URL(this._url);
+        let keysToRemove = []
+        for (const [key, ] of wmtsUrl.searchParams) {
```

Comment:
```suggestion
        for (const key of Object.keys(wmtsUrl.searchParams)) {
```

Reply:
It does not work, searchParams does not contain keys. It is an array of key values.